README.md

```diff
@@ -2,28 +2,52 @@
 
 Docsy is a [Hugo](https://gohugo.io/) theme for technical documentation sets, providing simple navigation, site structure, and more.
 
-This is not an officially supported Google product. This project is currently maintained.
+This is not an officially supported Google product. This project is actively being maintained.
 
 
-## Installation and prerequisites
+## Prerequisites and installation overview
 
-You need a recent version of Hugo to build sites using this theme (preferably 0.45+). If you install from the [release page](https://github.com/gohugoio/hugo/releases), make sure to get the `extended` Hugo version which supports SCSS. Alternatively, on macOS you can install Hugo via Brew.
+The following high-level list includes the basic prerequisites for using Docsy in your site. 
 
-You'll also need `PostCSS` to create the final CSS assets when building your site. You can install it locally with:
+<!-- TODO: Update to docsy.dev URL -->
+See the 
+[Docsy Getting Started Guide](https://docsydocs.netlify.com/docs/getting-started/) for a complete list of the
+prerequisites and details about the various installation options.
 
-```
-npm install
-````
+To use the Docsy theme in your site, you must meet the following prerequisites:
 
-To use a local version of the theme files, clone the repo using:
+- Install a recent release of the Hugo "extended" version (we recommend version 0.53 or later). If you install from the 
+  [release page](https://github.com/gohugoio/hugo/releases), you must ensure to download the `hugo_extended` version 
+  which supports SCSS.
 
-```
-git clone --recurse-submodules --depth 1 https://github.com/google/docsy.git
-```
+- Get the Docsy theme:
 
+  - (Recommended) Start with the Docsy Example project and use Docsy as a submodule. If you clone the Docsy Example site, 
+    you can modify and customize a pre-configured site, into your own Docsy themed site. 
+    [Learn more...](https://github.com/google/docsy-example)
+  
+  - Add docsy to your exsiting site. You can either add Docsy as a submodule, or clone the Docsy repo into your site.
+    See the complete list of [Docsy installation options](https://docsydocs.netlify.com/docs/getting-started/) .
+  
+    For example, to clone Docsy into the `theme` directory of your project, you run the following command from 
+    your project's root directory:
+    
+    ```
+    git clone --recurse-submodules --depth 1 https://github.com/google/docsy.git themes/docsy
+    ```
```

Comment:
I don't think this "for example..." bit is necessary anymore given your helpful  restructuring - we haven't given an example of how to do the "recommended" thing, why are we showing them how to do the less recommended thing, especially as there's a link to docs.

Reply:
good point